source/docs/articles/wordpress/site-networks/managing.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Managing WordPress Site Networks
+description: Learn how to use the Pantheon Workflow on Sites Networks
+---
+## Using the Pantheon Workflow with Multisite
+
+The Pantheon workflow enables you to easily create Dev and Test environments from live data. When using WordPress Multisite on Pantheon, you can still sync the database and all static files between environments. However, for greatest precision, performing a search and replace operation on the database is best done at the command line with Terminus and WP-CLI.
+
+### Basic Search and Replace on WordPress Multisite with a Single Site
+
+Say, for instance, you’ve just converted a WordPress environment in Dev to Multisite. You want to reuse this environment in Test, so you deploy your `wp-config.php` changes, and clone your database and files from Dev to Test. The last step is to perform a search and replace on the database.
+
+Because you just have one site on your Multisite Network, you can perform the search and replace in one operation:
+
+```bash
+terminus wp --site=handbuilt-site-network --env=test search-replace dev-handbuilt-site-network.pantheon.io test-handbuilt-site-network.pantheon.io --url=dev-handbuilt-site-network.pantheon.io --network
+```
+
+**Pro Tip**: Include the --dry-run flag to get a preview of the changes without destructively transforming the database.
+
+Before you run the command, let’s review each argument:
+
+`terminus wp` tells Terminus to perform a WP-CLI command
+`--site=handbuilt-site-network --env=test` are associative arguments interpreted by Terminus to know which site and which environment to run the WP-CLI command on
+`search-replace dev-handbuilt-site-network.pantheon.io test-handbuilt-site-network.pantheon.io` are three positional arguments interpreted by WP-CLI. WP-CLI interprets them as “run the search and replace command; use `dev-handbuilt-site-network.pantheon.io` as the string to search for; use `test-handbuilt-site-network.pantheon.io` as the value to replace it with.
+`--url=dev-handbuilt-site-network.pantheon.io` is an associative argument to set the context with which to load WordPress. Because the database still has the Dev URL stored, WordPress thinks it should be the Dev domain until the search and replace has been performed
+`--network` is a flag to perform the search and replace across the entire network
+
```

Comment:
Is `--network` required if there is only one site existing within the installation?